rosidl_typesupport_c/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,99 @@
+This document is a declaration of software quality for the `rosidl_typesupport_c` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `rosidl_typesupport_c` Quality Declaration
+
+The package `rosidl_typesupport_c` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 4 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy
+
+### Version Scheme
+
+`rosidl_typesupport_c` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+However, it is not yet at a stable version, i.e. `>= 1.0.0`.
```

Comment:
Just a note that this line should be updated when we bump the package to 1.0 for Foxy.

Reply:
Ok, for now this will just reflect the current status.